rclpy/test/test_subscription.py

```diff
@@ -48,7 +49,13 @@ def make_mock_subscription(namespace, topic_name, cli_args=None):
     ('/example/topic', 'ns', '/example/topic'),
 ])
 def test_get_subscription_topic_name(topic_name, namespace, expected):
-    sub = make_mock_subscription(namespace, topic_name)
+    node = Node('node_name', namespace=namespace, cli_args=None)
```

Comment:
@ahcorde why copy `make_mock_subscription` implementation here? Same below.

Reply:
the node only exists in the mock function, then the object will be destroy becuase there is no reference to the node. I can return the node too to keep the object alive.